modules_artemis/__HELPERS/mobs.dm

```diff
@@ -0,0 +1,409 @@
+	if(!tails_list_human.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/tails/human, tails_list_human)
+	if(!tails_list_lizard.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/tails/lizard, tails_list_lizard)
+	if(!snouts_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/snouts, snouts_list)
+	if(!horns_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/horns, horns_list)
+	if(!ears_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/ears, horns_list)
+	if(!frills_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/frills, frills_list)
+	if(!spines_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/spines, spines_list)
+	if(!legs_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/legs, legs_list)
+	if(!body_markings_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/body_markings, body_markings_list)
+	if(!wings_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/wings, wings_list)
+	if(!tentacles_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/tentacles, tentacles_list)
+
+	//For now we will always return none for tail_human and ears.
+	return(list("mcolor" = pick("FFFFFF","7F7F7F", "7FFF7F", "7F7FFF", "FF7F7F", "7FFFFF", "FF7FFF", "FFFF7F"), "tail_lizard" = pick(tails_list_lizard), "tail_human" = "None", "wings" = "None", "snout" = pick(snouts_list), "horns" = pick(horns_list), "ears" = "None", "frills" = pick(frills_list), "spines" = pick(spines_list), "body_markings" = pick(body_markings_list), "legs" = "Normal Legs"))
+
+/proc/random_hair_style(gender)
+	switch(gender)
+		if(MALE)
+			return pick(hair_styles_male_list)
+		if(FEMALE)
+			return pick(hair_styles_female_list)
+		else
+			return pick(hair_styles_list)
+
+/proc/random_facial_hair_style(gender)
+	switch(gender)
+		if(MALE)
+			return pick(facial_hair_styles_male_list)
+		if(FEMALE)
+			return pick(facial_hair_styles_female_list)
+		else
+			return pick(facial_hair_styles_list)
+
+/proc/random_unique_name(gender, attempts_to_find_unique_name=10)
+	for(var/i=1, i<=attempts_to_find_unique_name, i++)
+		if(gender==FEMALE)
+			. = capitalize(pick(first_names_female)) + " " + capitalize(pick(last_names))
+		else
+			. = capitalize(pick(first_names_male)) + " " + capitalize(pick(last_names))
+
+		if(i != attempts_to_find_unique_name && !findname(.))
+			break
+
+/proc/random_unique_lizard_name(gender, attempts_to_find_unique_name=10)
+	for(var/i=1, i<=attempts_to_find_unique_name, i++)
+		. = capitalize(lizard_name(gender))
+
+		if(i != attempts_to_find_unique_name && !findname(.))
+			break
+
+/proc/random_unique_plasmaman_name(attempts_to_find_unique_name=10)
+	for(var/i=1, i<=attempts_to_find_unique_name, i++)
+		. = capitalize(plasmaman_name())
+
+		if(i != attempts_to_find_unique_name && !findname(.))
+			break
+
+/proc/random_skin_tone()
+	return pick(skin_tones)
+
+var/list/skin_tones = list(
```

Comment:
defining a proc using a variable before you define the variable triggers me

(pls move the list definition to above the proc)